app/controllers/users_controller.rb

```diff
@@ -10,7 +10,7 @@ def create
       }
       render json: { 'logged_in': true }
     else
-      render json: { errors: user.errors.full_messages }
+      render json: { errors: user.errors.full_messages }, status: :conflict
```

Comment:
Can we actually check if there's a conflict for this? And not just assume its in the email